### PR TITLE
feat: OCI 1.1 support

### PR DESCRIPTION
**Pull request by Two-Hearts**
As the default notation cli version of setup has been upgraded to v1.2.0, creating PR to support OCI 1.1 with new field `force_referrers_tag`.

Tests are also included in the PR.

This PR resolves part of issue #76.

### CODE REVIEW

**Review by duffney — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "notation",
-    "version": "1.0.0",
+    "version": "1.2.0",
```
Comment (shizhMSFT) [inline package.json:3]: Can you create an issue to track creating a release checklist?

File: src/setup.ts [hunk 2]
```diff
@@ -57,8 +59,14 @@ async function setup(): Promise<void> {
...
 }
-  
-export = setup;
+
+// notationCLIVersion returns the semantic version of notation CLI
+export const notationCLIVersion = async (): Promise<string> => {
+    const {stdout: stdout} = await exec.getExecOutput('notation', ['version']);
+    let versionOutput = stdout.split("\n");
+    return String(semver.clean(versionOutput[2].split(":")[1].trim()));
+}
```
Comment (shizhMSFT) [inline src/setup.ts:68]: Can you have a better way to extract `version` as the `notation version` may have more fields in the future and they can be before the `version` field.

**Review by shizhMSFT — APPROVED** [review]
LGTM with suggestions

File: package.json (hunk 1 above)
Comment (Two-Hearts) [reply package.json:3]: Issue created: https://github.com/notaryproject/notation-action/issues/83

File: .github/workflows/e2e-test-verify.yml [hunk 3]
```diff
@@ -87,7 +89,7 @@ jobs:
             target_artifact_reference: ${{ env.target_artifact_reference }}
             trust_policy: ./tests/e2e/verify/trustpolicy/trustpolicy.json
             trust_store: ./tests/e2e/verify/truststore
-
+
```
[the remark this answers is not in the document]
Comment (Two-Hearts) [reply .github/workflows/e2e-test-verify.yml:92]: Removed.

File: src/setup.ts (hunk 2 above)
Comment (Two-Hearts) [reply src/setup.ts]: Updated.

**Review by shizhMSFT — APPROVED** [review]
LGTM

**Pull request merged**